src/Matcher/Pattern/TypePattern.php

```diff
@@ -69,10 +78,28 @@ public function matchExpanders($value)
     }
 
     /**
-     * @return null|string
+     * {@inheritdoc}
      */
     public function getError()
     {
         return $this->error;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function hasExpander(string $expanderName)
+    {
+        foreach ($this->expanders as $expander) {
+            if (!$this->expanderInitializer->hasExpanderDefinition($expanderName)) {
```

Comment:
if we use constants for expander name , there is no need to use `getName` neither `is($name)` I'll revert changes to `PatternExpander` interface